.lighthouserc.yml

```diff
@@ -0,0 +1,17 @@
+ci:
+  assert:
+    assertions:
+      categories:accessibility: [error, minScore: 0.94]
+      categories:best-practices: [error, minScore: 1]
+      categories:performance: [error, minScore: 0.68]
+      categories:seo: [error, minScore: 0.92]
+    includePassedAssertions: true
+  collect:
+    numberOfRuns: 1
+    startServerCommand: http-server dist-ssr --brotli -c31536000
```

Comment:
Yes there are multiple degrees, depending on these goals:
1. Do we want to stick with a single tool to preview prerendered pages?
  * This would be easy: `http-server dist-ssr`
2. Should the preview be suited for performance measurements?
  * This would require the `precompress` step. We could add it to `prerender-preview`, which would slow this script down a bit. Or we'd have two `prerender-preview` scripts?

Reply:
I like the current setup with `npm run prerender-precompress` and `npm run prerender-preview`.